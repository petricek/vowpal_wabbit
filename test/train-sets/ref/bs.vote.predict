0.000000 0.000000 0.000000
0.000000 0.000000 0.129255
0.000000 0.000000 0.148772
0.000000 0.000000 0.060840
0.000000 0.000000 0.058303
0.000000 0.000000 0.081537
-1.000000 0.054089 0.092737
-1.000000 0.154763 0.197300
-1.000000 0.070440 0.119735
-1.000000 0.106002 0.156970
-1.000000 0.131082 0.249384
-1.000000 0.114773 0.254007
-1.000000 0.052804 0.232697
-1.000000 0.139756 0.277597
-1.000000 0.090664 0.180884
-1.000000 0.221884 0.327061
-1.000000 0.165338 0.254794
-1.000000 0.209842 0.305762
-1.000000 0.340508 0.462637
-1.000000 0.120226 0.224193
-1.000000 0.073768 0.145120
-1.000000 0.220500 0.484307
-1.000000 0.190630 0.352775
-1.000000 0.182753 0.237213
-1.000000 0.106899 0.132471
-1.000000 0.227949 0.439784
-1.000000 0.128184 0.229064
-1.000000 0.180219 0.302672
-1.000000 0.228122 0.350704
-1.000000 0.204299 0.334774
-1.000000 0.314371 0.492843
-1.000000 0.126741 0.261167
-1.000000 0.124671 0.196431
-1.000000 0.251582 0.346041
-1.000000 0.290159 0.443080
-1.000000 0.229357 0.485388
-1.000000 0.231179 0.370752
-1.000000 0.260514 0.425360
-1.000000 0.322384 0.437226
-1.000000 0.364678 0.533837
-1.000000 0.177008 0.266921
-1.000000 0.241793 0.396669
-1.000000 0.285598 0.407842
-1.000000 0.193210 0.319297
-1.000000 0.237762 0.343675
-1.000000 0.174958 0.296000
-1.000000 0.300795 0.506584
-1.000000 0.288176 0.417867
-1.000000 0.114779 0.289452
-1.000000 0.138826 0.201798
-1.000000 0.214783 0.286147
-1.000000 0.262592 0.320855
-1.000000 0.345773 0.354915
-1.000000 0.287803 0.387301
-1.000000 0.340064 0.436494
-1.000000 0.511989 0.623278
-1.000000 0.382258 0.459053
-1.000000 0.288175 0.342293
-1.000000 0.394197 0.466037
-1.000000 0.294464 0.365842
-1.000000 0.274843 0.291871
-1.000000 0.370239 0.451945
-1.000000 0.263405 0.546204
-1.000000 0.172470 0.328874
-1.000000 0.384146 0.471825
-1.000000 0.210561 0.314986
-1.000000 0.290151 0.393025
-1.000000 0.519332 0.677885
-1.000000 0.308720 0.388894
-1.000000 0.326835 0.549774
-1.000000 0.361177 0.381201
-1.000000 0.344763 0.482737
-1.000000 0.186831 0.391690
-1.000000 0.260427 0.353950
-1.000000 0.215816 0.423719
-1.000000 0.263440 0.350049
-1.000000 0.186749 0.422403
-1.000000 0.334917 0.564390
-1.000000 0.140700 0.157153
-1.000000 0.155413 0.222536
-1.000000 0.214465 0.251913
-1.000000 0.221673 0.341357
-1.000000 0.452622 0.556624
-1.000000 0.237801 0.345560
-1.000000 0.184397 0.410450
-1.000000 0.232534 0.295388
-1.000000 0.205586 0.235662
-1.000000 0.363244 0.641292
-1.000000 0.513498 0.600989
-1.000000 0.205673 0.398527
-1.000000 0.103954 0.159302
-1.000000 0.232096 0.429181
-1.000000 0.638817 0.855750
-1.000000 0.144414 0.229953
-1.000000 0.382206 0.497457
-1.000000 0.260217 0.416374
-1.000000 0.412449 0.512699
-1.000000 0.197576 0.228019
-1.000000 0.270814 0.396705
-1.000000 0.259860 0.350019
-1.000000 0.516710 0.734798
-1.000000 0.402081 0.536439
-1.000000 0.310487 0.439211
-1.000000 0.395488 0.444034
-1.000000 0.324233 0.669387
-1.000000 0.436018 0.529132
-1.000000 0.144183 0.212978
-1.000000 0.301875 0.423716
-1.000000 0.151935 0.270446
-1.000000 0.313713 0.617612
-1.000000 0.486268 0.628787
-1.000000 0.283770 0.614956
-1.000000 0.369105 0.478522
-1.000000 0.251930 0.303008
-1.000000 0.299528 0.438022
-1.000000 0.365449 0.614935
-1.000000 0.237040 0.314758
-1.000000 0.393253 0.576619
-1.000000 0.227074 0.553198
-1.000000 0.348598 0.419655
-1.000000 0.253989 0.401551
-1.000000 0.423812 0.491729
-1.000000 0.379711 0.429375
-1.000000 0.333694 0.490340
-1.000000 0.475125 0.614723
-1.000000 0.297861 0.342370
-1.000000 0.301465 0.403506
-1.000000 0.426926 0.679017
-1.000000 0.152801 0.375067
-1.000000 0.284597 0.481419
-1.000000 0.282384 0.382172
-1.000000 0.473508 0.759282
-1.000000 0.152288 0.243092
-1.000000 0.236476 0.314583
-1.000000 0.343871 0.434590
-1.000000 0.175347 0.320161
-1.000000 0.474025 0.569365
-1.000000 0.331816 0.472708
-1.000000 0.561289 0.662703
-1.000000 0.285891 0.427921
-1.000000 0.416796 0.476442
-1.000000 0.203135 0.267721
-1.000000 0.595398 0.642266
-1.000000 0.649297 0.773475
-1.000000 0.402790 0.533114
-1.000000 0.398944 0.529411
-1.000000 0.384409 0.520133
-1.000000 0.754406 0.861520
-1.000000 0.274039 0.503595
-1.000000 0.303270 0.422431
-1.000000 0.304247 0.664980
-1.000000 0.248617 0.370770
-1.000000 0.517339 0.637698
-1.000000 0.429443 0.512155
-1.000000 0.538520 0.771671
-1.000000 0.493010 0.605456
-1.000000 0.697762 0.832035
-1.000000 0.495899 0.620651
-1.000000 0.574845 0.838479
-1.000000 0.437458 0.610670
-1.000000 0.421177 0.653228
-1.000000 0.236033 0.459578
-1.000000 0.310350 0.631562
-1.000000 0.624589 0.683267
-1.000000 0.326519 0.338183
-1.000000 0.354893 0.453203
-1.000000 0.709282 0.875874
-1.000000 0.334571 0.542989
-1.000000 0.427073 0.528986
-1.000000 0.435002 0.854465
-1.000000 0.582296 0.687556
-1.000000 0.316739 0.417971
-1.000000 0.203978 0.322191
-1.000000 0.321507 0.375389
-1.000000 0.165551 0.244836
-1.000000 0.476566 0.620723
-1.000000 0.294528 0.318210
-1.000000 0.715140 0.845894
-1.000000 0.556829 0.692226
-1.000000 0.695677 0.767937
-1.000000 0.427762 0.542853
-1.000000 0.381689 0.470247
-1.000000 0.437735 0.513902
-1.000000 0.303374 0.444996
-1.000000 0.201594 0.447449
-1.000000 0.335906 0.586852
-1.000000 0.566305 0.799865
-1.000000 0.551280 0.675520
-1.000000 0.227184 0.381761
-1.000000 0.535517 0.737854
-1.000000 0.462763 0.651277
-1.000000 0.208805 0.454696
-1.000000 0.196710 0.286818
-1.000000 0.271469 0.292503
-1.000000 0.259612 0.431565
-1.000000 0.448857 0.662813
-1.000000 0.410403 0.555561
-1.000000 0.276979 0.446122
-1.000000 0.252549 0.430267
-1.000000 0.471164 0.648070
