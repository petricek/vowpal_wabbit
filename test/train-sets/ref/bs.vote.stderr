final_regressor = models/bs.vote.model
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
predictions = bs.vote.predict
using no cache
Reading datafile = train-sets/0001.dat
num sources = 1
average    since         example     example  current  current  current
loss       last          counter      weight    label  predict features
0.333333   0.333333            3         3.0   0.0000   0.0000       57
0.333333   0.333333            6         6.0   1.0000   0.0000       31
0.636364   1.000000           11        11.0   0.0000  -1.0000       38
0.818182   1.000000           22        22.0   1.0000  -1.0000       88
0.909091   1.000000           44        44.0   0.0000  -1.0000       66
0.954023   1.000000           87        87.0   1.0000  -1.0000       61
0.977011   1.000000          174       174.0   1.0000  -1.0000       26

finished run
number of examples per pass = 200
passes used = 1
weighted example sum = 200
weighted label sum = 0
average loss = 0.98
best constant = 0
total feature number = 15482
